### PR TITLE
docs: Migrate link formats

### PR DESCRIPTION
**Pull request by ashleemboyer**
## What

- Migrates the formats of links in Markdown content as part of the [Docs Content Link Rewrites project](https://docs.google.com/document/d/1WaSyvoVPS-YCNiSPX0ynGpvc1gySEcpbrRoQVhimQCA/edit)

## Reviewing

- [ ] The updated links should point to the same pages
- [ ] The lists of "unrewriteable" links below should not contain any links we expect to be rewritten

These are all the links that were determined to be "unrewriteable" (AKA no rewrite needed to be applied based on the rewrite script's logic):

<details>
<summary>🌟 Expand for lists of links by file name</summary>

```JSON
{
  "website/docs/plugin/sdkv2/best-practices/depending-on-providers.mdx": [
    "https://github.com/hashicorp/terraform/issues/new",
    "https://github.com/hashicorp/terraform/tree/main/docs/plugin-protocol"
  ],
  "website/docs/plugin/sdkv2/best-practices/deprecations.mdx": [
    "#provider-attribute-removal",
    "#provider-attribute-rename",
    "#renaming-a-required-attribute",
    "#renaming-an-optional-attribute",
    "#renaming-a-computed-attribute",
    "#provider-data-source-or-resource-removal",
    "#provider-data-source-or-resource-rename",
    "#provider-attribute-removal",
    "#renaming-a-required-attribute",
    "#renaming-an-optional-attribute",
    "#renaming-a-computed-attribute",
    "#renaming-a-computed-attribute",
    "#renaming-an-optional-attribute",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema#ResourceData.GetOk",
    "#provider-attribute-removal",
    "#renaming-a-computed-attribute",
    "#renaming-a-required-attribute",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema#ResourceData.GetOk",
    "#provider-attribute-removal",
    "#renaming-an-optional-attribute",
    "#renaming-a-required-attribute",
    "#provider-attribute-removal",
    "#provider-data-source-or-resource-removal",
    "#provider-attribute-removal"
  ],
  "website/docs/plugin/sdkv2/best-practices/naming.mdx": [
    "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones"
  ],
  "website/docs/plugin/sdkv2/best-practices/sensitive-state.mdx": [
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema#Schema.Sensitive",
    "https://app.terraform.io/",
    "https://github.com/hashicorp/terraform/issues/516"
  ],
  "website/docs/plugin/sdkv2/best-practices/testing.mdx": [
    "#built-in-patterns",
    "#basic-test-to-verify-attributes",
    "#update-test-verify-configuration-changes",
    "#expecting-errors-or-non-empty-plans",
    "#regression-tests",
    "#table-of-contents",
    "#table-of-contents",
    "#table-of-contents",
    "#table-of-contents",
    "#table-of-contents",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestStep.ExpectNonEmptyPlan",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestStep.ExpectError"
  ],
  "website/docs/plugin/sdkv2/best-practices/versioning.mdx": [
    "http://semver.org/"
  ],
  "website/docs/plugin/sdkv2/debugging.mdx": [
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/plugin#ServeOpts.Debug"
  ],
  "website/docs/plugin/sdkv2/guides/terraform-0.12-compatibility.mdx": [
    "https://github.com/golang/go/wiki/Modules",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/customdiff#ComputedIf"
  ],
  "website/docs/plugin/sdkv2/guides/v1-upgrade-guide.mdx": [
    "https://github.com/hashicorp/terraform-plugin-sdk",
    "https://github.com/hashicorp/tf-sdk-migrator",
    "#deprecations",
    "https://github.com/hashicorp/tf-sdk-migrator",
    "#deprecations",
    "#deprecations",
    "https://pkg.go.dev/github.com/hashicorp/tf-sdk-migrator@v1.4.0/cmd/check#CheckSDKPackageImports",
    "https://stedolan.github.io/jq/",
    "https://en.wikipedia.org/wiki/Grep",
    "https://github.com/radeksimko/go-refs",
    "https://en.wikipedia.org/wiki/Sed",
    "https://github.com/radeksimko/terraform-provider-sdk-exposure",
    "https://github.com/hashicorp/terraform-provider-consul/pull/149/files",
    "https://github.com/hashicorp/terraform-provider-kubernetes/pull/620/files",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/httpclient#TerraformUserAgent",
    "https://pkg.go.dev/github.com/hashicorp/terraform@v0.14.7/httpclient#New",
    "https://github.com/hashicorp/terraform/tree/e1d0acda0b19be25ea96748896d3cd7117df955a/flatmap"
  ],
  "website/docs/plugin/sdkv2/guides/v2-upgrade-guide.mdx": [
    "https://github.com/hashicorp/terraform-plugin-sdk/blob/v2.0.0/CHANGELOG.md",
    "#dropped-support-for-terraform-0-11-and-below",
    "#version-2-of-the-module",
    "#a-new-acceptance-testing-driver",
    "#more-support-for-context-context",
    "#support-for-diagnostics",
    "#support-for-resource-level-and-field-level-descriptions",
    "#new-diagnostics-enabled-map-validators",
    "#support-for-debuggable-provider-binaries",
    "#support-for-provider-metadata",
    "#louder-helper-schema-resourcedata-set-errors-in-testing",
    "#more-consistent-usage-of-github-com-mitchellh-go-testing-interface",
    "#corrected-results-of-helper-acctest-randintrange",
    "#clearer-handling-of-nil-for-helper-resource-nonretryableerror-and-helper-resource-retryableerror",
    "#more-robust-handling-of-helper-schema-typeset-hashes",
    "#stronger-validation-for-helper-schema-schema-computed-fields",
    "#more-robust-validation-of-helper-schema-typemap-elems",
    "#more-robust-validation-of-helper-resource-testcheckresourceattrpair",
    "#more-robust-validation-of-test-sweepers",
    "#deprecation-of-helper-schema-existsfunc",
    "#deprecation-of-helper-schema-schemavalidatefunc",
    "#removal-of-helper-mutexkv-package",
    "#removal-of-helper-pathorcontents-package",
    "#removal-of-httpclient-package",
    "#removal-of-helper-hashcode-package",
    "#removal-of-the-acctest-package",
    "#removal-of-the-terraform-resourceprovider-interface",
    "#removal-of-the-helper-schema-provider-stopcontext-method",
    "#removal-of-helper-schema-resourcedata-setpartial",
    "#removal-of-the-helper-schema-provider-metareset-property",
    "#removal-of-deprecated-validation-functions",
    "#removal-of-helper-schema-schema-promotesingle",
    "#removal-of-helper-schema-resourcedata-unsafesetfieldraw",
    "#removal-of-helper-schema-resource-refresh",
    "#removal-of-helper-schema-schema-removed",
    "#removal-of-helper-encryption-package",
    "#removal-of-discouraged-variables-functions-types-and-interfaces",
    "https://www.hashicorp.com/blog/deprecating-terraform-0-11-support-in-terraform-providers/",
    "https://github.com/hashicorp/tf-sdk-migrator#tf-sdk-migrator-v2upgrade-migrate-from-sdkv1-to-sdkv2",
    "https://github.com/hashicorp/terraform/issues",
    "https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications",
    "https://github.com/hashicorp/terraform-plugin-sdk/issues/476",
    "https://github.com/hashicorp/terraform-plugin-sdk/issues",
    "https://github.com/hashicorp/terraform-plugin-sdk/issues"
  ],
  "website/docs/plugin/sdkv2/index.mdx": [
    "https://github.com/hashicorp/terraform-provider-scaffolding"
  ],
  "website/docs/plugin/sdkv2/logging/http-transport.mdx": [
    "https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/logging/transport.go",
    "https://pkg.go.dev/net/http#RoundTripper",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Resource.ReadContext",
    "https://pkg.go.dev/net/http#NewRequestWithContext",
    "https://pkg.go.dev/net/http#Request.WithContext",
    "https://en.wikipedia.org/wiki/Universally_unique_identifier",
    "https://developer.mozilla.org/en-US/docs/Web/HTTP/Method",
    "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-log/tflog",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging#NewLoggingHTTPTransport",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging#NewSubsystemLoggingHTTPTransport"
  ],
  "website/docs/plugin/sdkv2/resources/customizing-differences.mdx": [
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/customdiff"
  ],
  "website/docs/plugin/sdkv2/resources/import.mdx": [
    "#importer-state-function"
  ],
  "website/docs/plugin/sdkv2/resources/retries-and-customizable-timeouts.mdx": [
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#ResourceData.Timeout",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Resource.Timeouts"
  ],
  "website/docs/plugin/sdkv2/resources/state-migration.mdx": [
    "#terraform-v0-11-sdk-state-migrations",
    "#terraform-v0-12-sdk-state-migrations"
  ],
  "website/docs/plugin/sdkv2/schemas/index.mdx": [
    "https://github.com/hashicorp/terraform-plugin-sdk/tree/main/helper/schema",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Schema"
  ],
  "website/docs/plugin/sdkv2/schemas/schema-behaviors.mdx": [
    "https://pkg.go.dev/builtin#bool",
    "#default",
    "https://pkg.go.dev/builtin#bool",
    "https://golang.org/doc/effective_go#interfaces",
    "https://pkg.go.dev/builtin#bool",
    "https://pkg.go.dev/builtin#bool",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#SchemaDiffSuppressFunc",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#SchemaDefaultFunc",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#SchemaStateFunc",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#SchemaValidateFunc",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/validation",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#SchemaValidateDiagFunc",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/diag#Diagnostics",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
  ],
  "website/docs/plugin/sdkv2/schemas/schema-methods.mdx": [
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Schema"
  ],
  "website/docs/plugin/sdkv2/schemas/schema-types.mdx": [
    "https://pkg.go.dev/builtin#bool",
    "https://pkg.go.dev/builtin#int",
    "https://pkg.go.dev/builtin#float64",
    "https://pkg.go.dev/builtin#string",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation#IsRFC3339Time",
    "https://golang.org/doc/effective_go#maps",
    "https://golang.org/doc/effective_go#slices",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Set"
  ],
  "website/docs/plugin/sdkv2/testing/acceptance-tests/index.mdx": [
    "https://github.com/hashicorp/go-plugin",
    "https://github.com/hashicorp/terraform/tree/main/docs/plugin-protocol",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/internal/plugintest",
    "https://go.dev/",
    "#terraform-cli-installation-behaviors",
    "#environment-variables",
    "#requirements-and-recommendations",
    "https://pkg.go.dev/cmd/go/internal/test",
    "#command-line-workflow",
    "#github-actions-workflow",
    "https://www.gnu.org/software/make/manual/make.html#Makefiles",
    "https://github.com/",
    "https://github.com/features/actions",
    "https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization",
    "https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository",
    "https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions",
    "https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows",
    "https://github.com/actions/checkout",
    "https://github.com/actions/setup-go",
    "https://github.com/hashicorp/setup-terraform",
    "https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#ParallelTest",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#Test",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestStep.Config",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestStep.Config",
    "https://pkg.go.dev/os#TempDir",
    "https://pkg.go.dev/log",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#ParallelTest",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#ParallelTest",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#Test",
    "https://pkg.go.dev/testing#T.Parallel"
  ],
  "website/docs/plugin/sdkv2/testing/acceptance-tests/testcase.mdx": [
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCase",
    "https://pkg.go.dev/builtin#bool",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Provider",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestCheckFunc",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestStep",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestCase",
    "https://pkg.go.dev/testing#T"
  ],
  "website/docs/plugin/sdkv2/testing/acceptance-tests/teststep.mdx": [
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestStep",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCheckResourceAttr",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestMatchResourceAttr",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCheckResourceAttrPair",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCheckResourceAttrSet",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCheckNoResourceAttr",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestCheckTypeSetElemAttr",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestCheckTypeSetElemAttrPair",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestCheckTypeSetElemNestedAttrs",
    "https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/resource#TestCheckFunc"
  ],
  "website/docs/plugin/sdkv2/testing/index.mdx": [
    "https://github.com/terraform-providers"
  ],
  "website/docs/plugin/sdkv2/testing/unit-testing.mdx": [
    "https://pkg.go.dev/testing",
    "https://github.com/hashicorp/terraform-provider-aws/blob/f22ae122d8407672bd38951f80a2813b8b9af683/aws/structure_test.go#L930-L1027"
  ]
}
```

</details>

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203715368770954
  - https://app.asana.com/0/0/1203534392784698

### CODE REVIEW

File: website/docs/plugin/sdkv2/logging/index.mdx [hunk 1]
```diff
@@ -6,20 +6,20 @@ description: |-
...
 
 ## Filtering Log Output
 
-Learn how to [implement code in provider logic](/plugin/log/filtering) to omit logs or mask specific log messages and structured log fields.
+Learn how to [implement code in provider logic](/terraform/plugin/log/filtering) to omit logs or mask specific log messages and structured log fields.
 
 ## Log HTTP Transactions
 
-Learn how to [set up the Logging HTTP Transport](./logging/http-transport) to log HTTP Transactions with the [structured logging framework](/plugin/log).
+Learn how to [set up the Logging HTTP Transport](/terraform/plugin/sdkv2/logging/logging/http-transport) to log HTTP Transactions with the [structured logging framework](/terraform/plugin/log).
```
Comment (ashleemboyer) [inline website/docs/plugin/sdkv2/logging/index.mdx:25]: 🔴 `"set up the Logging HTTP Transport"` link is not correct

Comment (ashleemboyer) [reply website/docs/plugin/sdkv2/logging/index.mdx:25]: This `dev-portal` PR fixes the issue identified: [[preAdjustUrl] Handle with currentPath has a trailing slash](https://github.com/hashicorp/dev-portal/pull/1615)

Comment (bendbennett) [reply website/docs/plugin/sdkv2/logging/index.mdx:25]: Link for `[set up the Logging HTTP Transport]` should be `/terraform/plugin/sdkv2/logging/http-transport`

**Review by bendbennett — APPROVED** [review]
LGTM 👍 

Have supplied the correct link for `[set up the Logging HTTP Transport]`.

**Comment by ashleemboyer** [comment]
Hi @bendbennett! Sorry for any confusion, this PR isn't quite ready for review yet. It will be opened for review within the next few weeks though! 😊

**Review by bflad — APPROVED** [review]

**Pull request merged**